server/server.go

```diff
@@ -11,11 +12,14 @@ import (
 var echoService *echo.Echo
 
 func Main(args []string) {
+	setupLifeJourneyFile()
 	// Echo instance
 	echoService = echo.New()
 	service()
 }
-
+func setupLifeJourneyFile() {
```

Comment:
Not a big deal but adding a space before and after this function will make it easier to read